app/views/service-instances.html

```diff
@@ -0,0 +1,84 @@
+<project-header class="top-header"></project-header>
+<project-page>
+
+  <!-- Middle section -->
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header header-toolbar">
+        <div class="container-fluid">
+          <div class="page-header page-header-bleed-right page-header-bleed-left">
+            <h1>
+              Provisioned Services
+            </h1>
+          </div>
+          <div class="data-toolbar">
+            <div class="data-toolbar-filter">
+              <project-filter></project-filter>
+            </div>
+          </div>
+        </div>
+      </div><!-- /middle-header-->
+      <div class="middle-content">
+        <div class="container-fluid">
+          <alerts alerts="alerts"></alerts>
+          <div class="row">
+            <div class="col-md-12">
+              <table class="table table-bordered table-hover table-mobile table-layout-fixed">
```

Comment:
We've removed `table-hover` from our tables.

Reply:
Sure thing, think that's the only spot... but scanning back through the templates/views on this PR